docs/formatting.md

```diff
@@ -122,7 +122,7 @@ You shouldn't create ad-hoc string formats if you can avoid it. If you intend fo
 
 ### toFormat
 
-See [DateTime#toFormat](../class/src/datetime.js~DateTime.html#instance-method-toFormat) for the API signature. As a brief motivating example:
```

Comment:
I don't understand this. The link is supposed to be to the method signature for `toFormat`, not to the table of tokens...

Reply:
You're right. Changed to https://moment.github.io/luxon/docs/class/src/datetime.js~DateTime.html#instance-method-toFormat